openlr/openlr_match_quality/openlr_assessment_tool/CMakeLists.txt

```diff
@@ -19,6 +19,8 @@ set(SRC
   trafficmodeinitdlg.ui
 )
 
+set_source_files_properties(mainwindow.cpp PROPERTIES COMPILE_FLAGS "-Wno-deprecated-declarations")
```

Comment:
So, remove this line if we don't use -Werror. Let everybody see these warnings until fix :)